classes/webservice/WebserviceRequest.php

```diff
@@ -267,16 +267,16 @@ protected function getOutputObject($type)
         }
         $this->outputFormat = $type;
         switch ($type) {
-            case 'JSON':
-                require_once __DIR__ . '/WebserviceOutputJSON.php';
-                $obj_render = new WebserviceOutputJSON();
+        case 'JSON':
+            include_once __DIR__ . '/WebserviceOutputJSON.php';
```

Comment:
What is the reason of changing these lines?

Reply:
it's just remove spaces and replace require_once by include_once This is done by PrettyCI I think
now I recovery change include by required
thanks

Reply:
You shouldn't run prettyci directly, we are using a specific configuration for php-cs-fixer and these files are already checked by it.

`./vendor/bin/php-cs-fixer fix`

Reply:
ok there, I will
thanks a lot